src/helpers/DirectionalLightHelper.js

```diff
@@ -11,7 +11,7 @@ const _v3 = /*@__PURE__*/ new Vector3();
 
 class DirectionalLightHelper extends Object3D {
 
-	constructor( light, size, color ) {
+	constructor( light, size = 1, color ) {
```

Comment:
TBH, I don't like the idea of having default parameters used in that order because of https://github.com/mrdoob/three.js/pull/20586/files#r514965354. `light` is mandatory, `size` and `color` optional. Using default parameters just for `size` looks odd.

Reply:
I agree.